### DISCUSSION TITLE
mini.icons configuration help

### DISCUSSION THREAD

**Discussion by hunleyd** [nvim-mini/mini.nvim#1089] [Q&A]
Been using and loving the various mini modules for some time. Recently rewrote my Lazy config to be more consistent and readable. You can see my 'mini' section beginning on line [83](https://github.com/hunleyd/neovim/blob/main/lua/hunleyd/lazy.lua#L83). It works well. However, I cannot get mini.icons to work unless I add it separately (line [217](https://github.com/hunleyd/neovim/blob/main/lua/hunleyd/lazy.lua#L217)). I'm fairly certain I'm just being dumb and the config can be reworked to match the rest of my 'mini' section, but I'm at a loss. Anyone care to help a brother out?

**Comment by echasnovski** [comment]
Sorry, I don't have much knowledge of 'lazy.nvim' management.

My feeling here is that the whole 'echasnovski/mini.nvim' is lazy loaded in which case there might be issues with 'mini.icons'. The whole 217 line is a LazyVim's way to make 'mini.icons' properly lazy-loadable in a 'lazy.nvim' sense.

The suggestion would be to find a way to execute `require('mini.icons').setup()` specifically during startup without lazy loading it. Or use ['mini.deps'](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-deps.md) :)

**Reply by hunleyd** [reply]
Thanks @echasnovski 
I've been looking at moving to mini.deps actually. Just have to find enough 'free time'.
